faq/migration-tidb-faq.md

```diff
@@ -72,7 +72,7 @@ TiDB can provide services while Loader is running because Loader inserts the dat
 
 ### How to export the data in TiDB?
 
-Currently, TiDB does not support `select into outfile`. You can use the following methods to export the data in TiDB:
+You can use the following methods to export the data in TiDB:
 
 - See [MySQL uses mysqldump to export part of the table data](https://blog.csdn.net/xin_yu_xin/article/details/7574662) in Chinese and export data using mysqldump and the `WHERE` clause.
```

Comment:
This is a chinese article. We shouldn't use this link in our English docs.

Reply:
Fixed in [13db08f](https://github.com/pingcap/docs/pull/10219/commits/13db08f027dc03df14e14cc3d4087816357da4bd)